# Add shebang for the simple R script so one can run as binary
# November 30th 2018 by B. Hajgato (Free University Brussels - VUB)
--- mcgranahanlab-lohhla-61f4675cdee9/LOHHLAscript.R.orig	2018-11-05 17:55:44.000000000 +0100
+++ mcgranahanlab-lohhla-61f4675cdee9/LOHHLAscript.R	2018-11-30 13:29:13.148912146 +0100
@@ -1,3 +1,4 @@
+#!/usr/bin/env Rscript
 # before running
 # ml BEDTools/2.26.0-foss-2016b
 # ml SAMtools/1.3.1-foss-2016b
